pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/distributed.tsx

```diff
@@ -85,5 +85,41 @@ export default function (props: GraphDashboardProps) {
         }
       </Axis>
     </LineGraph>,
+
+    <LineGraph title="Node Liveness Heartbeat Latency: 99th percentile"
+      tooltip={`The 99th percentile of latency to heartbeat a node's internal liveness record over a 1 minute period.
+                              Values are displayed individually for each node on each node.`}>
```

Comment:
did you mean just "for each node" or "on each node" instead of both?

Reply:
It does read a little funny. I copied it from our other latency graph tooltips. I've updated all of them to just say "for each node".